lms/services/course.py

```diff
@@ -156,6 +170,23 @@ def _is_pre_sections(self, context_id):
             )
         )
 
+    def _get_copied_from_course(self, lti_params) -> Course | None:
+        """Return the course that the current one was copied from."""
+
+        history_params = [
```

Comment:
It's odd to have this whole loop and list but as I said this is very similar to the assignments one so that' easier.

I'll add other variables in follow up PRs that focus on one LMS at a time.